reference/config_files/settings.yml.rst

```diff
@@ -112,7 +112,7 @@ Here you can find a brief explanation of each of the architectures defined as ``
 
 - **x86_64**: The popular 64 bit x64 architecture.
 
-- **ppc64le**: The PowerPC 64 bit Big Endian architecture.
+- **ppc64be**: The PowerPC 64 bit Big Endian architecture.
```

Comment:
This doesn't seem a typo in the name, but maybe in the description. The current set of architectures defined in settings.yml is:

```
arch: [x86, x86_64, ppc32be, ppc32, ppc64le, ppc64, armv4, armv4i, armv5el, armv5hf, armv6, armv7, armv7hf, armv7s, armv7k, armv8, armv8_32, armv8.3, sparc, sparcv9, mips, mips64, avr, s390, s390x, asm.js, wasm, sh4le]
```

Reply:
Current version of file:

....
ppc64le: The PowerPC 64 bit Big Endian architecture.
ppc32: The PowerPC 32 bit architecture.
ppc64le: The PowerPC 64 bit Little Endian architecture.
ppc64: The PowerPC 64 bit Big Endian architecture.
...


Proposed change:
...
Current version of file:
ppc64be: The PowerPC 64 bit Big Endian architecture.
ppc32: The PowerPC 32 bit architecture.
ppc64le: The PowerPC 64 bit Little Endian architecture.
ppc64: The PowerPC 64 bit Big Endian architecture.
...

Note the order in the line you quoted from the settings.yml file.

Reply:
Does ppc64 == ppc64be?

Should ppc64be change to ppc32be?

There looks to be more changes to fix than just descriptions.